docs/survey-builder.spec.md

```diff
@@ -0,0 +1,42 @@
+# Spec
```

Comment:
What do you think about wiki-ing this stuff?

Reply:
Sure. Whatever works. Checking it into version control just seemed more evident.

Reply:
Wiki's are under version control, just a separate repo - check https://github.com/decrn/localsurvey.wiki.git